app/controllers/api/account.php

```diff
@@ -1064,11 +1065,10 @@
 
         $emailVariables = [
             'direction' => $locale->getText('settings.direction'),
-            /* {{user}} ,{{team}}, {{project}} and {{redirect}} are required in the templates */
+            /* {{user}} ,{{team}} and {{project}} are required in the templates */
             'user' => '',
             'team' => '',
-            'project' => $project->getAttribute('name'),
-            'redirect' => $url
+            'project' => $project->getAttribute('name')
```

Comment:
I don't think redirect should be removed here. It should be a variable and the final rendered email uses the redirect variable in the worker.